dask_jobqueue/sge.py

```diff
@@ -62,8 +65,16 @@ def __init__(self, queue=None, project=None, resource_spec=None, walltime=None,
             header_lines.append('#$ -q %(queue)s')
         if project is not None:
             header_lines.append('#$ -P %(project)s')
+
+        # Define resource specifications. There are only two cases allowed in this
+        # implementation: either specify using resource_spec, or specify using the rest of the
+        # kwargs.
```

Comment:
@lesteve would say that this comment is useless, the code is explicit :)

Reply:
Removing!